### PR TITLE
Ticket 1024 - Subscribe to alerts flow styling

### LINKED ISSUE #1024: Subscribe to alerts flow styling
Match/improve upon subscribe to alerts styling. PS. let's not forget about customColorTheme in this modal!

### PR DESCRIPTION
**Pull request by eyosef**
This PR styles the subscribe to alerts feature. Fixes #1024 

What it accomplishes;
- matches comps
- improved UI/UX - back/forward buttons, form validation

What it doesn't accomplish;
- pixel perfection (but it's close) - I noticed this feature leans on `display: block` on `PROD` so I implemented flex grid instead. Styled to align with comps closely

### CODE REVIEW

File: src/css/alertCarousel.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    p {
+      font-weight: 400;
+      line-height: 1.7;
+    }
+
+    ul {
+      display: grid;
+      list-style: none;
+      grid-template-columns: 8rem 8rem;
```
Comment (vaidotasp) [inline src/css/alertCarousel.scss]: you actually can achieve the "place content in two equally sized columns that take up all space" like this `grid-template-columns: 1fr 1fr`

File: src/js/components/leftPanel/dataPanel/subscribeToAlerts/NameYourSubscription.tsx [hunk 2]
```diff
@@ -36,35 +36,42 @@ const NameYourSubscription = (
...
+          </select>
+        </div>
+        <div className="buttons-wrapper">
+          <button
+            onClick={(): void => setPrevStep()}
+            className="orange-button custom esri-icon-left-arrow"
+          />
+          <button
+            onClick={(): void => setNextStep()}
+            disabled={subscriptionName.length ? false : true}
```
Comment (vaidotasp) [inline src/js/components/leftPanel/dataPanel/subscribeToAlerts/NameYourSubscription.tsx]: do we need that ternary? `subscription.length should evaluate to boolean already? or am I missing some edge case?

**Review by vaidotasp — CHANGES_REQUESTED** [review]
In addition to those two small things, let's make sure that subscription flow respects `customColorTheme` from the config. You can check out various other components that are listening to this for example `PrintReportButton.tsx`

File: src/js/components/leftPanel/dataPanel/subscribeToAlerts/NameYourSubscription.tsx (hunk 2 above)
Comment (eyosef) [reply src/js/components/leftPanel/dataPanel/subscribeToAlerts/NameYourSubscription.tsx]: `subscriptionName.length` would return `true` and disable the button when we need the button to only be disabled when `subscriptionName.length` is 0. I can refactor so it looks like this;

```typescript
          <button
            onClick={(): void => setNextStep()}
            disabled={subscriptionName.length === 0}
```

**Review by vaidotasp — CHANGES_REQUESTED** [review]
can we make all buttons/icons in the flow respect customColorTheme? that seems to be in line with what is in prod right now.
image
image
image

**Review by vaidotasp — APPROVED** [review]

**Pull request merged**